lib/db/mem.js

```diff
@@ -104,6 +112,11 @@ module.exports = function (log, error) {
       lastAccessTime: sessionToken.createdAt
     }
 
+    unverifiedTokens[tokenId] = {
+      tokenVerificationId: sessionToken.tokenVerificationId,
+      uid: sessionToken.uid.toString('hex')
```

Comment:
> I don't think it matters really, but I'm happy to change it if you feel strongly about it.

I would treat them the same and be consistent with how tokens are stored elsewhere to reduce gotchas. When I first [read this snippet](https://github.com/mozilla/fxa-auth-db-mysql/pull/132/files/16c9cc05baca6b4cc748d8f01118f3aa7fb23061#diff-6b5c42ced82313f1e160fb6ce884c580R334), I was going to point out an error in comparing a buffer to a string (t.uid and uid) until I realized t.uid was stored as a string.